data/validator/standard_string_test.go

```diff
@@ -510,6 +612,46 @@ var _ = Describe("StandardString", func() {
 				})
 			})
 
+			Context("NotExists", func() {
```

Comment:
unsure if there is a way around it but I feel I have seen this ( `Context(""NotExists", func() ...` ) test / check several times before. Same with `Empty`